app/gui/src/dashboard/components/AriaComponents/ContextualHelp/ContextualHelp.stories.tsx

```diff
@@ -0,0 +1,213 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { useState } from 'react'
+import { Button } from '../Button'
+import { Text } from '../Text'
+import { ContextualHelp } from './ContextualHelp'
+
+const meta = {
+  title: 'Components/ContextualHelp',
+  component: ContextualHelp,
+  parameters: {
+    layout: 'centered',
+  },
+  argTypes: {
+    placement: {
+      control: 'select',
+      options: [
+        'top',
+        'bottom',
+        'left',
+        'right',
+        'top start',
+        'top end',
+        'bottom start',
+        'bottom end',
+        'left top',
+        'left bottom',
+        'right top',
+        'right bottom',
+      ],
+    },
+    variant: {
+      control: 'radio',
+      options: ['help', 'info'],
+    },
+  },
+} satisfies Meta<typeof ContextualHelp>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    placement: 'top',
+    children: (
+      <div className="max-w-xs p-4">
+        <Text.Heading level={3} className="mb-2">
+          Help Title
+        </Text.Heading>
+        <Text>
+          This is a contextual help message that appears when the user hovers over the help icon.
+        </Text>
+      </div>
+    ),
+  },
+}
+
+export const Variants: Story = {
+  render: () => (
+    <div className="flex items-center gap-4">
+      <div className="flex flex-col items-center gap-2">
+        <Text variant="caption">Help Variant</Text>
+        <ContextualHelp placement="top">
+          <div className="max-w-xs p-4">
+            <Text.Heading level={3} className="mb-2">
+              Help
+            </Text.Heading>
+            <Text>This is the default help variant with a question mark icon.</Text>
+          </div>
+        </ContextualHelp>
+      </div>
+
+      <div className="flex flex-col items-center gap-2">
+        <Text variant="caption">Info Variant</Text>
+        <ContextualHelp variant="info" placement="top">
+          <div className="max-w-xs p-4">
+            <Text.Heading level={3} className="mb-2">
+              Information
+            </Text.Heading>
+            <Text>This is the info variant with an information icon.</Text>
+          </div>
+        </ContextualHelp>
+      </div>
+    </div>
+  ),
+}
+
+export const Placements: Story = {
+  render: () => {
+    const content = (
+      <div className="max-w-xs p-4">
+        <Text.Heading level={3} className="mb-2">
+          Placement Demo
+        </Text.Heading>
+        <Text>This popover demonstrates different placement options.</Text>
+      </div>
+    )
+
+    return (
+      <div className="grid grid-cols-3 gap-8 p-8">
+        <div className="flex justify-center">
+          <ContextualHelp placement="top start">{content}</ContextualHelp>
```

Comment:
with the notifications PR having been merged in, feel free to try `StoryVariants`!

https://github.com/enso-org/enso/blob/47e66da6772c9dd108506e86de31dfe718509b24/app/gui/src/dashboard/components/Icon/Icon.stories.tsx#L107-L120

https://github.com/enso-org/enso/blob/47e66da6772c9dd108506e86de31dfe718509b24/app/gui/src/dashboard/components/AriaComponents/Text/Text.stories.tsx#L36-L52

Reply:
feedback is welcome/appreciated